### PR TITLE
SC-8594–drag icon on the topic card is not visible

### PR DESCRIPTION
**Pull request by monaweitzenberg**
# Description
drag icon on the topic card is not visible; make it visible again

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-8594

## Changes
css - add color: $white

## Screenshots of UI changes
before 
Bildschirmfoto 2021-02-05 um 17 06 32

after:
Bildschirmfoto 2021-02-05 um 17 02 10

### CODE REVIEW

**Comment by murat-merdoglu-hpi** [comment]
Please add CHANGELOG.

File: static/styles/courses/course.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 .move-handle {
 	cursor: move;
 	white-space: nowrap;
+	color: $colorWhite;
```
Comment (mwolowyk-cap) [inline static/styles/courses/course.scss:7]: are you sure th move-handle used always on dark background?

**Review by Metauriel — APPROVED** [review]

File: static/styles/courses/course.scss [hunk 2]
```diff
@@ -40,6 +40,10 @@
 		}
 	}
 
+	.dropdown-toggle:hover {
+        color: $primaryColor !important;
+    }
+
```
Comment (mwolowyk-cap) [inline static/styles/courses/course.scss:46]: Is this part of the story? If so please avoid using important in css. You should have a proper css structure instead

**Review by mwolowyk-cap — APPROVED** [review]
👍

**Review by murat-merdoglu-hpi — APPROVED** [review]

**Pull request merged**